lab_1_classify_by_unigrams/main.py

```diff
@@ -35,6 +43,17 @@ def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
     In case of corrupt input arguments, None is returned
     """
 
+    if not isinstance(tokens, list) or len(tokens) == 0 or None in tokens:
+        return None
+    freq = {}
+    all_tokens = len(tokens)
+    for token in tokens:
+        if token in freq:
+            freq[token] += 1 / all_tokens
```

Comment:
can you count ` 1 / all_tokens` earlier?

Reply:
Done